### ISSUE TITLE
examples目录下sh run_train.sh训练的时候，报这个错误是什么情况 ClassRegistrar.h:65] Check failed: mapGet(type, creatorMap_, &creator) Unknown class type: cudnn_conv

### ISSUE THREAD

**Issue by bolt163** [PaddlePaddle/DeepSpeech#176]
机器上有GPU的， 是因为编译paddle的时候cmake传了   -DWITH_GPU=OFF的原因吗
~/DeepSpeech/examples/aishell> sh run_train.sh
-----------  Configuration Arguments -----------
augment_conf_path: conf/augmentation.config
batch_size: 64
dev_manifest: data/aishell/manifest.dev
init_model_path: None
is_local: 1
learning_rate: 0.0005
max_duration: 27.0
mean_std_path: data/aishell/mean_std.npz
min_duration: 0.0
num_conv_layers: 2
num_iter_print: 100
num_passes: 50
num_proc_data: 16
num_rnn_layers: 3
output_model_dir: ./checkpoints/aishell
rnn_layer_size: 1024
share_rnn_weights: 0
shuffle_method: batch_shuffle_clipped
specgram_type: linear
test_off: 0
train_manifest: data/aishell/manifest.train
trainer_count: 8
use_gpu: 1
use_gru: 1
use_sortagrad: 1
vocab_path: data/aishell/vocab.txt
------------------------------------------------
I0314 21:29:46.475132 40531 Util.cpp:166] commandline:  --use_gpu=1 --rnn_use_batch=True --log_clipping=True --trainer_count=8 
[INFO 2018-03-14 21:29:46,487 layers.py:2714] output for __conv_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2018-03-14 21:29:46,488 layers.py:3282] output for __batch_norm_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2018-03-14 21:29:46,489 layers.py:7454] output for __scale_sub_region_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2018-03-14 21:29:46,490 layers.py:2714] output for __conv_1__: c = 32, h = 41, w = 54, size = 70848
[INFO 2018-03-14 21:29:46,490 layers.py:3282] output for __batch_norm_1__: c = 32, h = 41, w = 54, size = 70848
[INFO 2018-03-14 21:29:46,491 layers.py:7454] output for __scale_sub_region_1__: c = 32, h = 41, w = 54, size = 70848
F0314 21:29:46.509068 40531 ClassRegistrar.h:65] Check failed: mapGet(type, creatorMap_, &creator) Unknown class type: cudnn_conv
*** Check failure stack trace: ***
    @     0x7f8b5119d99d  google::LogMessage::Fail()
    @     0x7f8b511a1b5c  google::LogMessage::SendToLog()
    @     0x7f8b5119d4c3  google::LogMessage::Flush()
    @     0x7f8b511a25ae  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f8b50ec67cb  paddle::Layer::create()
    @     0x7f8b50f6de48  _ZZN6paddle13NeuralNetwork4initERKNS_11ModelConfigESt8functionIFviPNS_9ParameterEEERKSt6vectorINS_19enumeration_wrapper13ParameterTypeESaISB_EEbENKUlRKNS_11LayerConfigEE_clESI_
    @     0x7f8b50f6ed4f  paddle::NeuralNetwork::init()
    @     0x7f8b50f7c7d9  paddle::MultiGradientMachine::MultiGradientMachine()
    @     0x7f8b50f80f8f  paddle::GradientMachine::create()
    @     0x7f8b5117a635  GradientMachine::createFromPaddleModelPtr()
    @     0x7f8b5117a81f  GradientMachine::createByConfigProtoStr()
    @     0x7f8b50e0a68d  _wrap_GradientMachine_createByConfigProtoStr
    @     0x7f8ba5726412  PyEval_EvalFrameEx
    @     0x7f8ba57281ce  PyEval_EvalCodeEx
    @     0x7f8ba57271f6  PyEval_EvalFrameEx
    @     0x7f8ba57281ce  PyEval_EvalCodeEx
    @     0x7f8ba57271f6  PyEval_EvalFrameEx
    @     0x7f8ba57281ce  PyEval_EvalCodeEx
    @     0x7f8ba56a38e8  function_call
    @     0x7f8ba5673dc3  PyObject_Call
    @     0x7f8ba568654f  instancemethod_call
    @     0x7f8ba5673dc3  PyObject_Call
    @     0x7f8ba56e0910  slot_tp_init
    @     0x7f8ba56d7328  type_call
    @     0x7f8ba5673dc3  PyObject_Call
    @     0x7f8ba57256c7  PyEval_EvalFrameEx
    @     0x7f8ba57281ce  PyEval_EvalCodeEx
    @     0x7f8ba57271f6  PyEval_EvalFrameEx
    @     0x7f8ba57281ce  PyEval_EvalCodeEx
    @     0x7f8ba57271f6  PyEval_EvalFrameEx
    @     0x7f8ba57281ce  PyEval_EvalCodeEx
    @     0x7f8ba57271f6  PyEval_EvalFrameEx
run_train.sh: line 34: 40531 Aborted                 CUDA_VISIBLE_DEVICES=0,1,2,3 python -u train.py --batch_size=64 --trainer_count=8 --num_passes=50 --num_proc_data=16 --num_conv_layers=2 --num_rnn_layers=3 --rnn_layer_size=1024 --num_iter_print=100 --learning_rate=5e-4 --max_duration=27.0 --min_duration=0.0 --test_off=False --use_sortagrad=True --use_gru=True --use_gpu=True --is_local=True --share_rnn_weights=False --train_manifest='data/aishell/manifest.train' --dev_manifest='data/aishell/manifest.dev' --mean_std_path='data/aishell/mean_std.npz' --vocab_path='data/aishell/vocab.txt' --output_model_dir='./checkpoints/aishell' --augment_conf_path='conf/augmentation.config' --specgram_type='linear' --shuffle_method='batch_shuffle_clipped'
Failed in training!

**Comment by zy486at189cn** [comment]
我也遇到了这个情况，我的GPU是gtx1060,是不是显缓不足？
